pages/glossary/focused.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Focused
+key: focused
+unambiguous: true
+objective: true
+---
+
+An element is focused when that element matches the [`:focus` pseudo-class](https://drafts.csswg.org/selectors-4/#focus-pseudo).
```

Comment:
This is generic (not specific for HTML), but would probably benefit from adding
"for HTML elements, see also https://html.spec.whatwg.org/multipage/semantics-other.html#selector-focus" which is much more detailed than the CSS version.

Reply:
Updated